Sources/Amplitude/AMPStorage.m

```diff
@@ -0,0 +1,103 @@
+//
+//  File.m
+//
+//
+//  Created by Dante Tam on 6/10/21.
+//
+
+#import <Foundation/Foundation.h>
+#import <Amplitude.h>
+#import "AMPStorage.h"
+
+@implementation AMPStorage
+
++ (NSString *)getAppStorageAmpDir:(NSString *)instanceName {
+    NSArray *paths = NSSearchPathForDirectoriesInDomains(NSApplicationSupportDirectory, NSUserDomainMask, YES);
+    NSString *path = [paths firstObject];
+    NSString *bundleIdentifier = [[NSBundle mainBundle] bundleIdentifier];
+    return [NSString stringWithFormat:@"%@/%@/%@", path, bundleIdentifier, instanceName];
```

Comment:
What if `instanceName` is nil?

Reply:
I just tested it when I made the PR. It goes to `$default_instance`